aw_qt/trayicon.py

```diff
@@ -128,12 +128,7 @@ def add_module_menuitem(module):
 def exit_dialog():
     # TODO: Do cleanup actions
     # TODO: Save state for resume
-    options = QMessageBox.Yes | QMessageBox.No
-    default = QMessageBox.No
-    answer = QMessageBox.question(None, "ActivityWatch", "Are you sure you want to quit?", options, default)
-
-    if answer == QMessageBox.Yes:
-        exit()
+    exit()
```

Comment:
I'd remove the `edit_dialog` function entirely, just need to change line 76 and 78. 

No idea what those TODO comments are about either, so feel free to remove them.